From 11d2c65456a71762744d2ffce321579249c70752 Mon Sep 17 00:00:00 2001
From: Fabian Groffen <grobian@gentoo.org>
Date: Sat, 19 Dec 2020 16:38:36 +0100
Subject: [PATCH] [ld64] unwinddump: include stdlib for exit(3)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix warning
../darwin-xtools/ld64/src/other/unwinddump.cpp:1174:6:
error: ‘exit’ was not declared in this scope; did you mean ‘_exit’?

Signed-off-by: Fabian Groffen <grobian@gentoo.org>
---
 ld64/src/other/unwinddump.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git ld64/src/other/unwinddump.cpp ld64/src/other/unwinddump.cpp
index d77e5d9..168ea67 100644
--- ld64/src/other/unwinddump.cpp
+++ ld64/src/other/unwinddump.cpp
@@ -27,6 +27,7 @@
 #include <sys/mman.h>
 #include <stdarg.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <fcntl.h>
 #include <unistd.h>
 #include <errno.h>
-- 
2.38.1

